spec/models/asset_spec.rb

```diff
@@ -114,7 +114,7 @@
     end
 
     it 'should handle umlauts and non english characters in the filename' do
-      filename = 'müppets.mp3'.mb_chars.normalize
+      filename = 'müppets.mp3'.unicode_normalize(:nfc)
```

Comment:
I guess this changed in https://github.com/sudara/alonetone/commit/5242f8eebd3fdedaee8d209181159d4f2b0c59e6 but I'm kind of curious why we are manually normalizing at all here. Is it because of s3 needs?

Reply:
An editor (or any other data processor like a database) may store Unicode in composed or decomposed form, so we can't rely on what we are getting from the source code to get a stable representation for a string.

```ruby
'é'.chars #=> ["é"]
'é'.unicode_normalize(:nfd).chars #=> ["e", "́"]
```

When you do a string comparison in a spec you will get a false negative if any processor force the string to another normalization form.

```ruby
'é'.unicode_normalize(:nfd) == 'é'.unicode_normalize(:nfc) #=> false
```

I can't remember if this was an issue here, but I assume that's why I added it.